.github/workflows/ci.yml

```
@@ -836,6 +810,23 @@ jobs:
         with:
           submodules: true
+
+      - name: Configure Docker daemon for mirror.gcr.io
+        run: |
+          echo '{"registry-mirrors": ["https://mirror.gcr.io"]}' | sudo tee /etc/docker/daemon.json
+          sudo systemctl restart docker
+          docker logout || echo "No Docker Hub login found"
+
+      - name: Start E2E services
+        run: docker compose -f e2e/compose.e2e.yml up -d
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)